### DISCUSSION TITLE
Downloadable Alpine Linux distro for WSL2

### DISCUSSION THREAD

**Discussion by ibeketov** [microsoft/WSL#6276] [General]
Currently is possible to manually download Windows Subsystem for Linux distro packages for all supported systems except Alpine Linux ( [Downloading distros](https://docs.microsoft.com/en-us/windows/wsl/install-manual#downloading-distros)).  AL is a great minimal distro especially as a hosting system for Docker, so it would be great if it will be possible to download it manually.

**Comment by 8549** [comment]
https://www.microsoft.com/en-us/p/alpine-wsl/9p804crf0395?activetab=pivot:overviewtab

**Comment by Biswa96** [comment]
Here is another popular one https://github.com/yuk7/AlpineWSL

**Comment by ibeketov** [comment]
> https://www.microsoft.com/en-us/p/alpine-wsl/9p804crf0395?activetab=pivot:overviewtab
Thank you but this is the distro installation from MS Store but it would be great to have the downloadable version.

> Here is another popular one https://github.com/yuk7/AlpineWSL
Thank you it looks great but I would prefer to have the official distro from MS

**Reply by nbetcher** [reply]
@ibeketov No need to actually install via the Microsoft Store GUI in Windows in order to install the WSL version of Alpine Linux. The AppX Bundle is available here: https://bit.ly/3BJQTdQ

Once the AppX Bundle file is downloaded, simply follow the last, directly-linked section of this page: https://bit.ly/3775diI

**Note:** The AppX Bundle download link is hosted directly from Microsoft.com and was extracted by means of https://store.rg-adguard.net. **However, the version of Alpine Linux for WSL posted on the Microsoft Store is NOT official! It is maintained by a separate third-party.**

Enjoy!

**Reply by theraw** [reply]
why do you need to use bit.ly links and not direct source?

**Reply by nbetcher** [reply]
> why do you need to use bit.ly links and not direct source?

Why does _**anyone**_ ever _**need**_ to use URL shortening services? Besides, I did it because the URLs are extremely long and I didn't want to drown out the remainder of the comment with unnecessary URL length. Does that meet with your standards? 😂

**Reply by kairusds** [reply]
I'm a bit skeptical as to why the supposed `download` link has non-matching URLs on the certificate info.

**Comment by therealkenc** [comment]
#6190
